.gitattributes

```diff
@@ -1 +1,2 @@
 *.go text eol=lf
+*.tf text eol=lf
```

Comment:
This is to avoid test failures (in CI) on Windows caused by converted line endings in test config files.